doc/ref/mloop.xml

```diff
@@ -961,6 +961,41 @@ or <C>brk></C> or <C>brk_<A>nn</A>></C> prompt.
 </Description>
 </ManSection>
 
+<ManSection>
+<Func Name="GAP_EXIT_CODE" Arg='ret'/>
+
+<Description>
+A <C>GAP_EXIT_CODE</C> sets the return value which will be used when
+&GAP; exits. This may be an integer, or a boolean (where
+<K>true</K> is interpreted as 0, and
+<K>false</K> is interpreted as 1.
+<P/>
+</Description>
+</ManSection>
+
+<ManSection>
+<Func Name="QUIT_GAP" Arg='[ret]'/>
+
+<Description>
+A <C>QUIT_GAP</C> acts similarly to the keyword <C>quit</C>. It exits
+&GAP; cleanly, calling any function installed using <C>InstallAtExit</C>.
+The optional argument will be passed to <C>GAP_EXIT_CODE</C>.
+<P/>
+</Description>
+</ManSection>
+
+<ManSection>
+<Func Name="FORCE_QUIT_GAP" Arg='[ret]'/>
+
+<Description>
+A <C>FORCE_QUIT_GAP</C> is similar to <C>QUIT_GAP</C>, except it ignores any
+functions installed with <C>InstallAtExit</C>, or any other functions
+normally run at GAP exit, and exits GAP immediately.
+The optional argument will be passed to <C>GAP_EXIT_CODE</C>.
```

Comment:
Would it be better to put a proper cross-reference here (and above), replacing `<C>GAP_EXIT_CODE</C>` by `<Ref Func="GAP_EXIT_CODE"/>` ?